.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: nim
```

Comment:
https://docs.travis-ci.com/user/languages/

I don't think Nim is available.

Reply:
https://github.com/nim-lang/Nim/wiki/BuildServices

That shows several options. I think `choosenim` is the slowest. Docker+sudo in TravisCI should work well, but we need the `devel` branch. I don't know how to do that. (I've asked here: https://forum.nim-lang.org/t/4801#30053 )

We might be better off with CircleCI.

Reply:
I'm working on TravisCI in #2  now ...